packages/babel-types/src/generated/helpers.js

```diff
@@ -0,0 +1,1024 @@
+// @flow
```

Comment:
Maybe a comment saying 

```js
// This file is auto-generated! run `node scripts/generateTypeHelpers` to update.
```

Reply:
Good idea, the generating script should insert something like this.

Reply:
Yes will definitely add this. For now just waiting for some more general feedback or if anyone sees a problem with this solution that I haven't thought of.

Reply:
Imho that's a great move even if only for documentation purposes. It will be easier to look on the generated file and see what exactly is available, what API it has etc than on the generating file (which kinda is available on the master right now).